packages/backend/src/api/job/jobRouter.ts

```diff
@@ -0,0 +1,299 @@
+import { OpenAPIRegistry } from '@asteasolutions/zod-to-openapi';
+import express, { Request, Response, Router } from 'express';
+import { z } from 'zod';
+
+import { createApiResponse } from '../../api-docs/openAPIResponseBuilders';
+import { handleServiceResponse } from '../../common/utils/httpHandlers';
+import { GetCategorySchema, GetJobSchema, GetJobViews, JobCategorySchema, JobSchema } from './jobModel';
+import { jobService } from './jobService';
+
+export const jobRegistry = new OpenAPIRegistry();
+
+jobRegistry.register('Job', JobSchema);
+
+export const jobRouter: Router = (() => {
+  const router = express.Router();
+
+  jobRegistry.registerPath({
+    method: 'get',
+    path: '/jobs',
+    tags: ['Job'],
+    responses: createApiResponse(z.array(JobSchema), 'Success'),
+  });
+
+  router.get('/', async (req: Request, res: Response) => {
+    const { pageIndex, pageSize, categoryId, levelId, companyName } = req.query;
```

Comment:
Get for required jobs -> validation rules. Flow as follows:
Route: 
create GetJobRequest object
Validate object
if validation fails -> return 400
else -> send to service